docs/speaker.rst

```diff
@@ -0,0 +1,34 @@
+Speaker **V2**
+**************
+
+.. py:module:: speaker
+
+The micro:bit **V2** has a built-in speaker located on the rear of the board.
+
+.. image:: speaker.png
+    :width: 300px
+    :align: center
+    :height: 240px
+    :alt: speaker on rear of micro:bit V2
+
+By default sound output will be via the edge connector on pin 0 and the
+built-in speaker **V2**. You can connect wired headphones or a speaker to
+pin 0 and GND on the edge connector to hear the sounds.
+
+The speaker can be turned off or on using the ``speaker`` object.
```

Comment:
I probably wouldn't call it an object as we are not really letting users create instances, and it might not end up being a module either (that will depend on the implementation).

We should just say that it can be turned on or off using the functions listed here.

Reply:
Done